modules/library/coverage/src/main/java/org/geotools/coverage/grid/io/imageio/geotiff/GeoTiffIIOMetadataDecoder.java

```diff
@@ -1,5 +1,5 @@
 /*
- * (c) 2004 Mike Nidel
+ * (c) 2004 - 2016 Mike Nidel
```

Comment:
@georgekankava please leave the original copyright header date range (there are two). Please only update the OSGeo copyright header (lower down) as this is the entity that now owns the rights in this work and thus your contribution. The previous copyright headers are retained for provenance.

It would be easier if the OSGeo header was at the top, but that is not your problem.